### PR TITLE
Add new pip installation to RAPIDS-Colab install script

### PR DESCRIPTION
**Pull request by taureandyernv**
- [x] Adding new script to install pip into env-check.py
- [x] Added warnings to not continue conda based approach

test colab notebook https://colab.research.google.com/drive/1RU2D0OfUp9vkpPnZGN1pSUrVtf_HcvPz?usp=sharing

### CODE REVIEW

**Comment by taureandyernv** [comment]
closes #63
closes #66

File: colab/env-check.py [hunk 1]
```diff
@@ -15,11 +28,47 @@
...
+    else:
+      print(line.rstrip())
+  output = subprocess.Popen(["pip install cupy-cuda11x"], shell=True, stderr=subprocess.STDOUT, 
+      stdout=subprocess.PIPE)
+  for line in io.TextIOWrapper(output.stdout, encoding="utf-8"):
+    if(line == ""):
+      break
+    else:
+      print(line.rstrip())
+  print("""
```
Comment (beckernick) [inline colab/env-check.py:58]: Why do we need to do this in subprocesses?

Are we able to just link to a fresh Colab notebook that does the necessary steps in a cell?

**Review by beckernick — CHANGES_REQUESTED** [review]

File: colab/env-check.py (hunk 1 above)
Comment (taureandyernv) [reply colab/env-check.py:58]: We are able to, BUT encapsulating the RAPIDS install in a set of script is an easy way to keep all content across all colab notebooks using the rapids-colab template up to date, despite numerous, behind-the-scenes changes.  If a user took an existing rapids-colab notebook that used the template for the install, from even 2 years ago, they would still get RAPIDS installed once this is updated.  If anything changes in the pip install, we change it here, and the user can still use that same template.

**Comment by reouvenzana** [comment]
Hello, I just tried your test colab notebook. The installation process is much faster than before, but I can't import any modules other than cudf.

When trying to import cuML, I get a ModuleNotFoundError.

**Comment by taureandyernv** [comment]
Fix for @reouvenzana 's issue was that I actually had to uncomment out the installs of cuml and cugraph (was testing basic install).  we're good now and I'm merging

**Pull request merged**